__tests__/index.test.js

```diff
@@ -264,3 +264,42 @@ describe("GET /api/reviews/:review_id/comments", () => {
       });
   });
 });
+
+describe("POST /api/reviews/:review_id/comments", () => {
+  test("adds a comment to database with username and body and returns the added comments", () => {
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send({ username: "bainesface", body: "test-body" })
+      .expect(201)
+      .then(({ body }) => {
+        const { comment } = body;
+        expect(comment).toEqual(
+          expect.objectContaining({
+            body: "test-body",
+            votes: 0,
+            author: "bainesface",
+            review_id: 5,
+            created_at: expect.any(String),
+          })
+        );
+      });
+  });
+  test("returns 404 for valid ID not in database", () => {
+    return request(app)
+      .post("/api/reviews/1000/comments")
+      .send({ username: "bainesface", body: "test-body" })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("ID or user not found");
+      });
+  });
+  test("returns 404 for usernames not in database", () => {
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send({ username: "not-a-user", body: "test-body" })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("ID or user not found");
+      });
+  });
```

Comment:
couple more tests to add - what is there was an empty comment body? 
if they entered an invalid article_id